backend/api/search.py

```diff
@@ -44,7 +44,7 @@ def search_simple():
     count = request.args.get('count', type=int)
     if count is None:
         count = 10
-    save = request.args.get('save', type=str)
+    save = json.loads(request.args.get('save', type=str))
```

Comment:
wouldn't json.loads(None) or json.loads("") throw an exception? should handle the case that 'save' is not submitted